druid/src/window.rs

```diff
@@ -252,8 +246,10 @@ impl<T: Data> Window<T> {
             }
         }
 
-        if let Some(cursor) = cursor {
+        if let Some(cursor) = &widget_state.cursor {
             self.handle.set_cursor(&cursor);
+        } else if matches!(event, Event::MouseMove(..)) {
+            self.handle.set_cursor(&Cursor::Arrow);
```

Comment:
It would be nice if it would only set the cursor when it actually changed. I think it was part of the nice to haves.
You could probably just compare the `Rc` since that should stay the same if the cursor doesn't change.

Reply:
I think there are cases where you have to set the cursor because it was invalidated some other way, for example when the window loses focus. My personal feeling is that it is possible to track those other state changes but simpler not to, so set it every time.

Reply:
Yeah, there's an issue open about this (#112) but I didn't feel like taking it on.